source/reference/operator/aggregation/geoNear.txt

```diff
@@ -204,8 +204,147 @@ When using :pipeline:`$geoNear`, consider that:
 - Starting in version 4.2, :pipeline:`$geoNear` no longer has a default
   limit of 100 documents.
 
-Example
--------
+- Starting in MongoDB 5.1, the ``near`` parameter supports the 
+  :ref:`let option <geoNear_let_example>` and
+  :ref:`bound let option <geoNear_bounded_let_example>`.
+
+Examples
+--------
+
+.. _geoNear_let_example:
+
+$geoNear with the ``let`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example:
+
+- The ``let`` option is used to set an array value of 
+  ``[-73.99279,40.719296]`` to the variable ``$pt``.
```

Comment:
What is the significance of the name "pt"? Is it short for "point", or is it an acronym of some kind?

Reply:
Its just the variable name for the point array.

Reply:
Thanks for clarifying. fwiw I don't love this as a variable name because it wasn't immediately clear to me what the significance of it was, but we can leave it alone.